dir.traversal.targets

```diff
@@ -39,7 +39,7 @@
         <UndefineProperties Condition="'%(Project.OSGroup)'==''">%(Project.UndefineProperties);OSGroup</UndefineProperties>
       </Project>
       <Project>
-        <UndefineProperties Condition="'%(Project.Extension)'!='.builds' and '%(Project.Extension)'!='.proj'">%(Project.UndefineProperties);TestTFMs;FilterToOSGroup;FilterToTestTFM;DefaultBuildAllTarget;SerializeProjects;BuildAllOSGroups</UndefineProperties>
+        <UndefineProperties Condition="'%(Project.Extension)'!='.builds' and '%(Project.Extension)'!='.proj'">%(Project.UndefineProperties);TestTFMs;FilterToOSGroup;FilterToTestTFM;DefaultBuildAllTarget;BuildAllProjects;SerializeProjects;BuildAllOSGroups</UndefineProperties>
```

Comment:
@weshaggard Note this change I had to do to avoid bin clashes. I'm not sure how this wasn't a problem before, but basically what I'm seeing is that some pkgprojs have P2P references to the ref project (Build doesn't pass BuildAllProjects) and also have a P2P reference to the src\<library>.builds file which references the ref project and passes BuildAllProjects=true